GraphQL-Schema-Design.md

```diff
@@ -6,9 +6,8 @@
 
 # How to model our graph
 
-* As with most GraphQL advice, our schema should strive to always be backwards compatible.
-  Thus careful consideration is needed about naming (be as unambiguous as possible) and the shape of the data returned
-  by a field.
+* As with most GraphQL advice, our schema should strive to always be backwards compatible. Thus careful consideration is
```

Comment:
I'm gonna move the number from 120 -> 110 so that there's padding for the `+` and `-` on here

Reply:
Wait, so in the context of MD prettier only counts the text for the width? That’s confusing.

Reply:
This was exactly 120:

<img width="1185" alt="screen shot 2017-11-21 at 9 23 19 am" src="https://user-images.githubusercontent.com/49038/33077368-a68305e4-ce9d-11e7-83f7-947e2942548f.png">

So rather than leave the occasional ugly orphaned small word in the editor because a diff adds a prefix of `+ ` and `- ` - I'd rather just reduce our line length a bit to accomodate

Reply:
Oooh, I misread `+` and `-` as being about creating an unordered list.